homeassistant/components/aladdin_connect/model.py

```diff
@@ -5,12 +5,26 @@
 from typing import TypedDict
 
 
-class DoorDevice(TypedDict):
-    """Aladdin door device."""
+class GarageDoorData(TypedDict):
+    """Aladdin door data."""
 
     device_id: str
     door_number: int
     name: str
     status: str
-    serial: str
-    model: str
+    link_status: str
+    battery_level: int
+
+
+class GarageDoor:
```

Comment:
I am not sure what the added benefit of this class is. It just wraps the class above and only sets an extra unique_id.